lib/bandit/http2/adapter.ex

```diff
@@ -81,6 +81,7 @@ defmodule Bandit.HTTP2.Adapter do
           File.stream!(path, [], 2048)
           |> Enum.reduce(adapter, fn chunk, adapter ->
             chunk(adapter, chunk)
+            adapter
```

Comment:
See comments on the larger review - this should be updated to an Enum.each iteration and comments added to the effect that the chunk return value is awkward to use here.

Reply:
I changed it to `Enum.each/2` and added a comment. 
Please take a quick look if the comment is correct and sufficient enough.

Reply:
I changed it to `Enum.each/2` and added a comment. 
Please take a quick look if the comment is correct and sufficient enough.